### DISCUSSION TITLE
help request: 四层IP白名单加动态代理

### DISCUSSION THREAD

**Discussion by feeops** [apache/apisix#6909] [Q&A]
### Description

这边有一个需求

有一堆IP白名单，适配了四层代理，是四层，不是七层

同一个转发路由，如果客户端IP地址是在IP白名单里面，就转发到本机的2222端口
如果不在里面，则转发到本机的2223端口

有灵活的办法可以实现吗？

### Environment

apsix最新版

**Comment by tzssangglass** [comment]
1. this is not problem of. APISIX, so I'm converting it into a discussion;
2. pls use English on public chanel;

you can try config `remote_addr` in the form of `CIDR`, see: https://github.com/apache/apisix/blob/master/docs/zh/latest/admin-api.md#stream-route
